packages/client/vitest.config.js

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    exclude: ['test/(integration|cli|sim)/**/*.spec.ts'],
```

Comment:
Why this line?  it caused the Client-CLI test to crash

Reply:
Sorry, I should have come back to this and hasn't gotten to it. This config sets up the unit tests correctly. Otherwise, vitest will just run everything that looks a test since it doesn't respect glob patterns from the CLI. We can update the integration and sim test npm scripts separately so they only target the specific folders. Will do that this morning.